bin/test-node.sh

```diff
@@ -12,6 +12,8 @@ else
 fi
 
 if [ $TYPE = "integration" ]; then
+    node bin/down-server.js 3010 & export SLOW_SERVER_PID=$!
```

Comment:
this still mentions SLOW :)

Reply:
Thank you for catching this!

Reply:
looks good to me, wanna squash this into your first commit? It wouldn’t make much sense to retain as a separate commit once this hits `master`.